apps/ui/src/routes/default.ts

```diff
@@ -40,9 +41,18 @@ export default [
 
   {
     path: '/create',
-    redirect: { name: 'create', params: { protocol: 'snapshot-x' } }
+    redirect: { name: 'space-create-snapshot-x' }
+  },
+  {
+    path: '/create/snapshot-x',
+    name: 'space-create-snapshot-x',
+    component: SpaceCreateSnapshotX
+  },
+  {
+    path: '/create/snapshot',
+    name: 'space-create-snapshot',
+    component: SpaceCreateSnapshot
   },
```

Comment:
All `spaceChildrenRoutes` use prefix `space-` but these two routes are not inside `space`, maybe better to use `create-snapshot-space` and `create-snapshot-x-space`

Just in case if someone use `name.startsWith('space-')`  like here https://github.com/snapshot-labs/sx-monorepo/blob/master/apps/ui/src/components/AppLink.vue#L24-L26

Reply:
Maybe use `create-space-snapshot` `create-space-snapshot-x` then